# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linux/ppc-radeon-qw-pm.patch
# Copyright (C) 2008 - 2019 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- linux-5.1/drivers/video/fbdev/aty/radeon_pm.c.orig	2019-05-11 11:05:49.676261522 +0000
+++ linux-5.1/drivers/video/fbdev/aty/radeon_pm.c	2019-05-11 11:07:11.132261185 +0000
@@ -2201,7 +2201,6 @@
 	radeon_pm_m10_enable_lvds_spread_spectrum(rinfo);
 }
 
-#if 0 /* Not ready yet */
 static void radeon_reinitialize_QW(struct radeonfb_info *rinfo)
 {
 	int i;
@@ -2503,7 +2502,6 @@
 	cgc |= 0x10000;
 	OUTREG(CUR_OFFSET, 0);
 }
-#endif /* 0 */
 
 #endif /* CONFIG_PPC_OF */
 
@@ -2849,12 +2849,10 @@
 			rinfo->reinit_func = radeon_reinitialize_M10;
 			rinfo->pm_mode |= radeon_pm_off;
 		}
-#if 0 /* Not ready yet */
 		if (!strcmp(rinfo->of_node->name, "ATY,BlueStoneParent")) {
 			rinfo->reinit_func = radeon_reinitialize_QW;
 			rinfo->pm_mode |= radeon_pm_off;
 		}
-#endif
 		if (of_node_name_eq(rinfo->of_node, "ATY,ViaParent")) {
 			rinfo->reinit_func = radeon_reinitialize_M9P;
 			rinfo->pm_mode |= radeon_pm_off;
